docs/src/pages/concepts/layouts.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Layouts
+caption: Learn about how layouting is handled in Marigold
+---
+
+Having a comprehensive collection of reusable components for common tasks (like buttons, cards and form fields) is great but it's not enough to build a whole application. An essential part is missing: components to construct and layout an application pages. Usually there are only a handful components dedicated for arranging and placing component in a page.
+
+A typically solution is to provide a `<Flex>` or `<Grid>` component for creating arbitrary layouts. However, in order to use those components efficiently developers need to have an intimate knowledge of `flex-box` and `grid`, including all their related properties.
+
+Although this gives app developers incredible freedom, it will also leave the burden on them to research and implement necessary layout patterns, while at the same time make them reusable and responsive. Ever heard of the [Flexbox Holy Albatross](https://heydonworks.com/article/the-flexbox-holy-albatross/)? Me neither.
+
+## Layout Foundation
+
+Because layout patterns are such an essential part of building great applications, Marigold comes with a set of layout components that will cover almost, if not all, necessary patterns.
```

Comment:
so almost all or all? 😁

Reply:
Both! Would you like to change it?

Reply:
what's the difference? :D i think it sounds like " Oh it's all there for you, or it's almost all there " :D i think if i read it: what is it.. all or not all? :D  i hope you understand me :D